tools/squidpy/squidpy.xml

```diff
@@ -0,0 +1,154 @@
+<tool id="squidpy" name="squidpy" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" python_template_version="3.5" profile="21.05">
+    <description> Squidpy framework for spatial omics analysis </description>
+    <macros>
+        <token name="@TOOL_VERSION@">1.2.2</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="1.2.2">squidpy</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        mkdir visium_out &&
+        tar -xzf ${visium} -C visium_out --strip-components=1 &&
+        python '$__tool_directory__/scanpy_preprocess.py'
+            -v visium_out
+            -c V1_Human_Lymph_Node_filtered_feature_bc_matrix.h5
+            #if $genome:
+                -g ${genome}
+            #end if
+            #if $library:
+                -l ${library}
+            #else:
+                -l 'None'
+            #end if
+            -s spatial/tissue_hires_image.png 
+            -x "${prefix}"
+            -m ${min_counts}
+            -M ${max_counts}
+            -p ${pct_counts}
+            -e ${min_cells}
+            -n ${n_top_genes}
+        && python '$__tool_directory__/squidpy_run.py'
+         -a squidpy_input.h5ad
+        #if $library:
+            -l ${library}
+        #else:
+            -l 'None'
+        #end if
+    ]]></command>
+    <inputs>
+        <!-- CHANGE FORMAT LATER -->
+        <param argument="--visium" type="data" format="visium.tar.gz" label="Visium dataset tarball"/>
```

Comment:
A user probably does not know what a tarball is. Just Visum dataset might be ok.